versions/3.0.4.md

```diff
@@ -1621,19 +1621,67 @@ An `encoding` attribute is introduced to give you control over the serialization
 A single encoding definition applied to a single schema property.
 See [Appendix B](#dataTypeConversion) for a discussion of converting values of various types to string representations.
 
-##### Fixed Fields
+Properties are correlated with `multipart` parts using the `name` parameter to `Content-Disposition: form-data`, and with `application/x-www-form-urlencoded` using the query string paramter names.
+In both cases, their order is implementation-defined.
+
+###### Common Fixed Fields
+
+These fields MAY be used either with or without the RFC6570-style serialization fields defined in the next section below.
+
+Field Name | Type | Description
+---|:---:|---
+<a name="encodingContentType"></a>contentType | `string` | The `Content-Type` for encoding a specific property. The value is a comma-separated list, each element of which is either a specific media type (e.g. `image/png`) or a wildcard media type (e.g. `image/*`). Default value depends on the property type as shown in the table below.
+<a name="encodingHeaders"></a>headers | Map[`string`, [Header Object](#headerObject) \| [Reference Object](#referenceObject)] | A map allowing additional information to be provided as headers. `Content-Type` is described separately and SHALL be ignored in this section. This property SHALL be ignored if the request body media type is not a `multipart`.
+
+This object MAY be extended with [Specification Extensions](#specificationExtensions).
+
+The default values for `contentType` are as follows:
+
+Property Type | Property Format | Default `contentType`
+------------- | --------------- | ---------------------
+`string` | `binary` | `application/octet-stream`
+`string`, `number`, `integer`, or `boolean` | _n/a_ | `text/plain`
```

Comment:
Do we have to consider `type: null` here?

Reply:
This is 3.0.4 so it's `nullable: true` and I wasn't sure if that should be handled differently as it's not a `type` keyword thing in this version.

We could change this from the value of keyword `type` to the JSON data model type (string, number, boolean, null) rather than the `type` + `nullable` keyword.

In any event, I suspect in 3.0 it is supposed to be determined by the non-`null` type.  Hmm... we'll have to figure that out for 3.1 as no one has ever asked what happens for a dual (or more) typed field.

Reply:
@ralfhandl the most recent force-push adds a commit that adds some discussion of `null` and reformats the table a bit.